.gitignore

```diff
@@ -150,9 +151,13 @@ test/write_*
 
 # Autogenerated docs
 docs/user_api/_autosummary/
-
 docs/notebook-examples.txt
-
+docs/*-examples-gallery.txt
+docs/user-guide/oQU120.data.nc
+docs/user-guide/oQU120.grid.nc
+docs/user-guide/oQU480.data.nc
+docs/user-guide/oQU480.grid.nc
+docs/user-guide/psi_healpix.nc
```

Comment:
I don't think we have netCDF files in the docs/user-guide dir, and probably these shouldn't be here. Are these from your local?